packages/next/server/config.ts

```diff
@@ -340,6 +340,19 @@ function assignDefaults(userConfig: { [key: string]: any }) {
         if (!item.defaultLocale) return true
         if (!item.domain || typeof item.domain !== 'string') return true
 
+        const defaultLocaleDuplicate = i18n.domains?.find(
+          (altItem) =>
+            altItem.defaultLocale === item.defaultLocale &&
+            altItem.domain !== item.domain
+        )
+
+        if (defaultLocaleDuplicate) {
+          console.warn(
```

Comment:
Should this be `console.error()` instead?

Reply:
We mainly use `console.warn` in the checks so went with that but an error is thrown here after this log https://github.com/vercel/next.js/blob/9cbc4aacd50d0e79be7b8178eec9219457514da9/packages/next/server/config.ts#L366